### DISCUSSION TITLE
Vote to retire @dumez-k from Dependencies Maintainer team

### DISCUSSION THREAD

**Discussion by dumez-k** [paketo-buildpacks/feedback#107] [Dependencies Team]
Hey team, with my increased responsibilities on the TBS side, along with @sophiewigmore  and @brayanhenao stepping up, this seems like a good time to shift to an emeritus status and remove myself from the Dependencies Maintainers team. Still will always be happy to help however if any context or anything is needed!

@paketo-buildpacks/dependencies-maintainers can I get a 👍 / 👎 ?

**Comment by ryanmoran** [comment]
With a super-majority vote to affirm, @dumez-k will be retired and moved to emeritus status.
